<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackboardInternalDialog</class>
 <widget class="QDialog" name="BlackboardInternalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blackboard internals</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons.qrc">
    <normaloff>:/Icons/car.png</normaloff>:/Icons/car.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="10,20">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0,0">
       <item>
        <widget class="ConfigFilter" name="FilterGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OnlyVisableCheckBox">
         <property name="text">
          <string>Only visable blackboard variable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="VariableListWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="InfosTabWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="BasicInfosTab">
        <attribute name="title">
         <string>Basic Infos</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="NamegroupBox">
           <property name="title">
            <string>Name:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="NameLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="DisplayNameGroupBox">
           <property name="title">
            <string>Display name:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="DisplayNameLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Value:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TextValueInput" name="ValueLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Vid:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="VidLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Data type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="DataTypeLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Unit:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="UnitLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="CommentGroupBox">
           <property name="title">
            <string>Comment:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPlainTextEdit" name="CommentPlainTextEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="DisplayTab">
        <attribute name="title">
         <string>Display</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Min Display Range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TextValueInput" name="MinLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Max Display Range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TextValueInput" name="MaxLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="WidthLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Prec:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PrecLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Step type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="StepTypeLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Step:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TextValueInput" name="StepLineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Color:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ColorLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Conversion type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ConversionTypeLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="ConversionGroupBox">
           <property name="title">
            <string>Conversion:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLineEdit" name="ConversionLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="AttachCounterTab">
        <attribute name="title">
         <string>Attach counter and flags</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Attach counter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="AttachCounterLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Unknown wait attach counter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="UnknownWaitAttachCounterLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="ProcessAttachCounterGroupBox">
           <property name="title">
            <string>Process attach counter</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTableWidget" name="ProcessAttachCounteTableWidget">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="rowCount">
               <number>64</number>
              </property>
              <property name="columnCount">
               <number>7</number>
              </property>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <row/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextValueInput</class>
   <extends>QLineEdit</extends>
   <header>TextValueInput.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigFilter</class>
   <extends>QGroupBox</extends>
   <header>ConfigFilter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BlackboardInternalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BlackboardInternalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
